src/platform/forms-system/src/js/helpers.js

```diff
@@ -701,3 +701,20 @@ export function transformForSubmit(
 
   return JSON.stringify(withoutViewFields, replacer) || '{}';
 }
+
+/**
+ * @param {Object} uiSchema A nested `uiSchema`
+ * @returns {Object} The closest `uiSchema` object that holds the `uiSchema` keys
+ */
+export const getNestedUISchema = uiSchema => {
```

Comment:
Do we have test coverage for this method to ensure the expected outcome?

Reply:
Yep, I just added some tests.